dist/ng-obiba-mica.js

```diff
@@ -3974,7 +3974,7 @@ angular.module('obiba.mica.search')
     };
 
     this.datasetPage = function(id, type) {
-      var dsType = (type.toLowerCase() === 'collection' ? 'collection' : 'harmonization') + '-dataset';
+      var dsType = (type.toLowerCase() === 'collected' ? 'collected' : 'harmonization') + '-dataset';
```

Comment:
It looks like harmonization-dataset end point has not been renamed to harmonized-dataset yet. This has to be done.